.github/workflows/lint.yml

```diff
@@ -0,0 +1,29 @@
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  check-format:
+    runs-on: ubuntu-latest
+    env:
+      # We need jq +1.7 because of jqlang/jq#1741.
+      JQ_VERSION: '1.7'
```

Comment:
According to https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md, ubuntu-24.04 has jq 1.7.1-3build1, so we can just use ubuntu-24.04 here and skip jq upgrade step.